packages/fether-react/src/Whitelist/Whitelist.js

```diff
@@ -69,6 +69,7 @@ class Whitelist extends Component {
     let db;
     try {
       db = await import(`../assets/tokens/${this.props.chainName}.json`);
+      db = db.default;
```

Comment:
Could be squashed into `db = await import(../assets/tokens/${this.props.chainName}.json).default;`